Jenkinsfile

```diff
@@ -0,0 +1,23 @@
+pipeline {
+    agent any
+
+    triggers {
+        githubPush()
+      }
+    tools {
+        maven 'MavenTest'
+    }
+    stages {
+        stage ('Initialize') {
+            steps {
+                sh 'mvn clean'
+            }
+        }
+
+        stage ('Build') {
+            steps {
+               sh 'mvn initialize test'
```

Comment:
Atom Hopper doesn't initialize. Also this should be a 'Test' phase.

Reply:
Done